### DISCUSSION TITLE
Unable to locate NSLog bootstrap script.

### DISCUSSION THREAD

**Discussion by 3rdcycle** [beeware/briefcase#678] [Q&A]
So, briefcase has been working beautifully for me in the past. But now with my latest build I get the error in the title when launching the app on macOS 11.6.

Any ideas on what could be wrong?

**Comment by freakboy3742** [comment]
The cause of this message is a side effect of a change in Briefcase v0.3.6. 

If you have a project that was generated with Briefcase v0.3.5 or earlier, you need to add "std-nslog==1.0.0" to your macOS and/or iOS requirements. 

This will be automatically added on any newly generated project; and we're in the process of modifying the app template to make this change transparent for existing projects.

Apologies for the confusion and inconvenience.

**Reply by freakboy3742** [reply]
FYI - I've just published an update to the app templates that will prevent this error. If you don't have `std-nslog` installed, you'll see an message in the log explaining why you don't see any more stdout/stderr output, and directing you to add `std-nslog` to your project requirements, but the app won't stop with the error dialog you've reported.

Apologies again for the inconvenience.
